recipe/build.sh

```diff
@@ -2,14 +2,18 @@
 
 set -o xtrace -o nounset -o pipefail -o errexit
 
-if [ $(uname) = Darwin ] ; then
+if [[ "${target_platform}" == osx-* ]]; then
   export RUSTFLAGS="-C link-args=-Wl,-rpath,${PREFIX}/lib"
 else
   export RUSTFLAGS="-C link-arg=-Wl,-rpath-link,${PREFIX}/lib -L${PREFIX}/lib"
 fi
 
+if [[ "${target_platform}" == "linux-64" ]]; then
+  export BUILD_ARGS="--features unwind"
+fi
```

Comment:
Think we need to export BUILD_ARGS for non x86_64 linux here: 

```suggestion
else
  export BUILD_ARGS=""
fi  
```